spec/lib/krikri/mapping_spec.rb

```diff
@@ -28,15 +28,52 @@
       expect(new_mapping.process_record(record)).to be_a klass
     end
 
-    context 'with parser' do
+    shared_context 'property declarations' do
       before do
-        target_instance = double
         allow(target_class).to receive(:new).and_return(target_instance)
         allow(target_instance).to receive(:my_property=).and_return('')
+        allow(parser).to receive(:parse).and_return(record)
         subject.my_property ''
       end
 
       subject { described_class.new(target_class, parser, *parser_args) }
+      let(:target_instance) { double('target instance') }
+    end
+    
+    describe 'error handling' do  
+      include_context 'property declarations'
+
+      before do
+        allow(target_instance).to receive(:error_property=).and_raise error
+        subject.error_property 'moomin'
```

Comment:
Does this get executed after `subject.process_record(record)`, below?
I'm just curious how this is is invoked.  Is this calling `:error_property=`?